src/main/java/org/fundacionjala/pivotal/pages/project/PageLogin.java

```diff
@@ -1,9 +1,14 @@
 package org.fundacionjala.pivotal.pages.project;
 
+import org.fundacionjala.pivotal.core.e2e.DriverManager;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.fundacionjala.pivotal.core.e2e.DriverManager'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)